src/containers/Post/PostList.js

```diff
@@ -14,7 +14,7 @@ class PostList extends Component {
     return (
       <div>
       <h2>Posts</h2>
-      {posts.items.map(post => (
+      {posts.items.length > 0 && posts.items.map(post => (
```

Comment:
No need to do this check, calling `map` on an empty array is completely fine. On the other hand, if `posts.items` is `null` or `undefined`, accessing items will fail. We don't need to be too defensive here, just making sure that non null `posts.items` is passed through the props is enough.